scrapy/aws.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import smtplib
+from email import encoders as Encoders
+from email.mime.base import MIMEBase
+from email.mime.multipart import MIMEMultipart
+from email.mime.nonmultipart import MIMENonMultipart
+from email.mime.text import MIMEText
+from email.utils import COMMASPACE, formatdate
+from io import BytesIO
+
+from scrapy.utils.misc import arg_to_iter
+
+class SESender(object): 
```

Comment:
```suggestion
class SESSender: 
```

Perhaps an even better name could be found for this class. Also, we're in py3 territory now, no need to inherit from `object`.